exercises/e001/results/propiro/outputBase32pipe.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```

Comment:
Put all those changes into `outputBase32faster.py` no need for 2 separate files and you have all the files in git history anyway.